koku/masu/util/aws/common.py

```diff
@@ -473,6 +585,45 @@ def _get_s3_objects(s3_path):
     return s3_resource.Bucket(settings.S3_BUCKET_NAME).objects.filter(Prefix=s3_path)
 
 
+def get_or_clear_s3_daily_csv_by_date(s3_path, start_date, manifest_id, context, request_id):
```

Comment:
I'm not sure I understand the purpose of this function. We do not create CSV files today, so there is nothing to delete...

Reply:
Correct but we will after this change and this also means if there are no files then we know we should process the full month :+1: